### PR TITLE
Added freegeek/models.py with Appointment, Station, Factory models. I…

### PR DESCRIPTION
**Pull request by mbrochmann**
…ssue #21

### CODE REVIEW

**Comment by mbrochmann** [comment]
Appointments should also have "slots", each with a "proficiency" requirement for the slot to be filled, and the "slots" should be able to be associated with a "user", or be "empty".

File: freegeek/models.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+from django.utils import timezone
+
+
+# Create your models here.                                                                                                                                               
+class Appointment(models.Model):
+    start_time = models.DateTimeField('start_time')
+    end_time = models.DateTimeField('start_time')
+    station = models.ForeignKey(Station, on_delete=models.CASCADE)
+    def __str__(self):
+        appointment_string = "Appointment: %s to %s at %s in %s" % (str(self.start_time), str(self.end_time), str(self.station), str(self.station.factory))
```
Comment (hobson) [inline freegeek/models.py]: close parens

File: freegeek/models.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        return appointment_string
+
+class Station(models.Model):
+    station_name = models.CharField(max_length=200)
+    factory = models.ForeignKey(Factory, on_delete=models.CASCADE)
+    def __str__(self):
+        return self.station_name
+
+class Factory(models.Model):
```
Comment (hobson) [inline freegeek/models.py]: change to Location or Address or Building

File: freegeek/models.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+    factory_name = models.CharField(max_length=200)
+    def __str__(self):
+        return self.factory_name
+
```
Comment (hobson) [inline freegeek/models.py]: Add docstrings

**Pull request merged**